docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -187,21 +189,52 @@ REST API and connect to a local validator:
 
   $ sudo -u sawtooth sawtooth-rest-api -v
 
-Running a Transaction Processor
-===============================
+Starting and Configuring the Transaction Processors
+===================================================
+
+This section describes how to start the IntegerKey and Settings family
+transaction processors, confirm that the REST API is running, and tell
+the validator or validator network to accept transactions from these
+transaction families.
 
 Transaction processors can be started either before or after the validator is
 started.
 
-The IntegerKey transaction processor is provided as a simple example of a
-transaction family, which can also be used for testing purposes.
+Supported Transaction Families
+------------------------------
+
+Sawtooth supports multiple languages for transaction processor development and
+includes additional transaction processors written in several languages.
+Sawtooth includes following transaction processors:
+
+* settings-tp - A Settings family transaction processor written in Python
```

Comment:
Missing the identity-tp

Reply:
Thanks for catching this!  I'd like to make the fix in a separate PR, because this text is also in the Docker and AWS procedures. (This content isn't new in this PR - I just moved it to a different spot in the procedure.)  If that's OK, I'll create a new PR to add the identity-tp to all three tutorials.